generators/app/templates/src/views/components/tab-navigation/tab-navigation.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+
+export const TabNavigation: FC = () => (
+  <nav>
+    <ul>
+      <li>
+        <button>Tab A</button>
```

Comment:
why are you not using your own button inside this component?

Reply:
I'd like to that in a separate PR, since I reeeeally want to get this shipped 😅